README.md

```diff
@@ -2,13 +2,6 @@
 
 [Fusebot](https://fusebot.io?utm_source=github.com&utm_medium=referral&utm_campaign=fusebot-readme&utm_content=intro) is the simplest way to quickly develop custom Slash Commands for Discord and Slack. Fusebot gives you a Node.js and npm development environment and takes care of running, scaling, and securing your Slash Commands. All you need to bring is your code and imagination.
 
-```javascript
-module.exports = async (ctx) => {
-  const r = await Superagent.get("https://sochain.com/api/v2/get_price/DOGE/USD");
-  await ctx.client.send(`:moneybag: DOGE price is ${r.body.data.prices[0].price} USD`);
-};
-```
-
```

Comment:
I think it is critical for the intro section to sell it enough for people to read on, especially if we plan to test the idea of using the readme as a landing page. To that end, a picture would be worth a thousand words. Maybe two over-imposed browser windows - one showing the Fusebit editor with code (coin to make it simple), the other the command actually being run in Slack/Discord? Something like this: https://github.com/tjanczuk/edge#before-you-dive-in